crates/iota-sdk-ffi/src/lib.rs

```diff
@@ -103,8 +103,8 @@
 //! [BCS]: https://docs.rs/bcs
 //! [RFC-5234]: https://datatracker.ietf.org/doc/html/rfc5234
 
+#![allow(clippy::wrong_self_convention)]
```

Comment:
what does it prevent us from doing?

Reply:
It's because of the uniffi methods where we want to take &self instead of self